mappings/net/minecraft/client/network/ServerInfo.mapping

```diff
@@ -16,6 +16,7 @@ CLASS net/minecraft/class_642 net/minecraft/client/network/ServerInfo
 	FIELD field_3763 local Z
 	FIELD field_39355 LOGGER Lorg/slf4j/Logger;
 	FIELD field_39356 chatPreview Lnet/minecraft/class_642$class_7484;
+	FIELD field_39436 chatPreviewEnabled Z
```

Comment:
I've mapped them as `chatPreviewEnabledByServer` on #3164 - let me check which one's better

Reply:
This one is temporary, unlike other ones - guess both are bad.